core/storage/base_model/gae_models.py

```diff
@@ -267,6 +282,23 @@ def get_all_commits(cls, page_size, urlsafe_start_cursor):
         return cls._fetch_page_sorted_by_last_updated(
             cls.query(), page_size, urlsafe_start_cursor)
 
+    @classmethod
+    def get_commit(cls, instance_id, version):
+        """Returns the commit corresponding to an instance id and
+        version number.
+
+        Args:
+            instance_id: str. The id of the particular construct, from which
+                this method is called.
+            version: int. The version number of the instance
+                after the commit.
+
+        Returns:
+            The commit with the instance id and version number.
```

Comment:
Add type info to beginning of description.

with the --> corresponding to the given

Reply:
Added the type.